package.json

```
@@ -22,9 +22,9 @@
     "@angular/platform-browser": "^16.0.0",
     "@angular/platform-browser-dynamic": "^16.0.0",
     "@angular/router": "^16.0.0",
-    "@jsonforms/angular": "3.2.0-beta.0",
-    "@jsonforms/angular-material": "3.2.0-beta.0",
-    "@jsonforms/core": "3.2.0-beta.0",
+    "@jsonforms/angular": "^3.2.0-beta.1",
+    "@jsonforms/angular-material": "^3.2.0-beta.1",
+    "@jsonforms/core": "^3.2.0-beta.1",
```

I prefer pinned version to make sure that they are on the same version. It's sufficient to do the change once we update to 3.2.0

```suggestion
    "@jsonforms/angular": "3.2.0-beta.1",
    "@jsonforms/angular-material": "3.2.0-beta.1",
    "@jsonforms/core": "3.2.0-beta.1",
```